### ISSUE TITLE
Double click not working on gnome 44

### ISSUE THREAD

**Issue by KayJay7** [ubuntu/gnome-shell-extension-appindicator#431]
On fedora 38 with gnome 44, double clicking a tray icon doesn't bring up the application windows.

Tried with telegram from flathub, steam from rpm and zapzap also from flathub. 

All of these used to get brought up on gnome 43, but aren't anymore on gnome 44.

The extension is the latest from extensions.gnome.org and gnome is version 44.0.

**Comment by kleidiss** [comment]
This is only a xorg issue , works fine on wayland

**Comment by KayJay7** [comment]
No, I'm on Wayland too

**Comment by EnverOsmanov** [comment]
After setting environment variable `G_MESSAGES_DEBUG=AppIndicator-and-KStatusNotifierItem-Support`, where should logs appear? I can't see any new logs in `journalctl`.

**Comment by Plarpoon** [comment]
I thought this was my issue alone but also on Fedora 38 and having the same exact issue

**Comment by Plarpoon** [comment]
> Again, no error logs, no way to handle this.
> 
> In case, export `G_MESSAGES_DEBUG=AppIndicator-and-KStatusNotifierItem-Support` so that gets picked by the shell, and that would cause more debugging bits to be written.

I am in an exclusively wayland environment, I added the env you have requested and I also cannot see anything new on journalctl. I even used the -f parameter to seek any single possible change or notification on the logs but absolutely nothing happened. If you have an alternative method I will be glad to actually operate it the way you request it. On what is the issue on the other hand I can try being a bit more descriptive.

This is an issue that present itself exclusively when trying to click on the icons of an XWayland application, I noticed my wayland apps don't suffer from this issue at all. Neither double clicking on it nor right clicking on it have any effect. The icon does show the graphical hint of when you click on something correctly, I think what @KayJay7 tried to say was that although that is the case the actual contextual menu that was supposed to appear does not appear at all.
